.github/workflows/health.yaml

```diff
@@ -141,6 +141,11 @@ on:
         default: "\"\""
         type: string
         required: false
+      health_yaml_name:
+        description: The name of the workflow file.
+        default: '""'
```

Comment:
Do we need a default here?

Reply:
It looks like below we default to checking for `health.yaml` or `health.yml`. Separately, it might be nice to figure out which of those extensions is more common, and only support that (`health.yaml`?).